composables/masto/relationship.ts

```diff
@@ -41,6 +41,7 @@ export async function toggleFollowAccount(relationship: mastodon.v1.Relationship
   if (unfollow) {
     if (await openConfirmDialog({
       title: i18n.t('confirm.unfollow.title'),
+      description: i18n.t('confirm.unfollow.description', [`@${account.acct}`]),
```

Comment:
Unlike the other confirm dialog, this message didn't have an account name place holder so I added one for a better dialog message.